src/com/intellij/idea/plugin/hybris/toolwindow/RemoteConnectionDialog.java

```diff
@@ -41,14 +41,17 @@
     private JTextField displayNameTextField;
     private OnOffButton sslButton;
     private JLabel sslLabel;
+    private JComboBox sslProtocol;
```

Comment:
If the SSL toggle is off we could disable sslProtocol UI configuration.

Reply:
Done